CONTRIBUTING.md

```diff
@@ -20,6 +20,11 @@ is strongly encouraged that you submit the patch to https://llvm.org/ directly.
 See [LLVM contribution guidelines](https://llvm.org/docs/Contributing.html)
 for more information.
 
+**NB**: A change in compiler and runtime should be accompanied with
+corresponding test change.
```

Comment:
```suggestion
corresponding test case.
```
?

Reply:
It can be more then one test case or test removal. "Test change" covers all possible modification "test case" don't.